main.cpp

```diff
@@ -179,10 +198,49 @@ int main(int argc, char* argv[]) {
     boat.position.x = i * 12.0f;
   }
 
-  std::shared_ptr<Model> testModel = std::make_shared<Model>("data/model/test");
+//  std::shared_ptr<Model> testModel = std::make_shared<Model>("data/model/test");
+  std::shared_ptr<Model> testModel = std::make_shared<Model>("/home/fox/Data/Projects/boat-game-proto-data/data/monaco");
+
+
+  Particles particles;
+
+  // Load particles from file
+  { 
+    FILE* f = fopen("/home/fox/box-constraint.obj", "rb");
```

Comment:
Needs fixing locally.

You can craft your own object using blender, including hinges and so on; see "Articulated Bodies" at http://graphics.cs.cmu.edu/nsp/course/15-869/2006/papers/jakobsen.htm.

This file is:

```
# Blender v2.79 (sub 0) OBJ File: ''
# www.blender.org
mtllib box-constraint.mtl
o Cube_Cube.001
v -1.000000 -1.000000 1.000000
v -1.000000 1.000000 1.000000
v -1.000000 -1.000000 -1.000000
v -1.000000 1.000000 -1.000000
v 1.000000 -1.000000 1.000000
v 1.000000 1.000000 1.000000
v 1.000000 -1.000000 -1.000000
v 1.000000 1.000000 -1.000000
l 3 1
l 1 2
l 2 4
l 4 3
l 7 3
l 4 8
l 8 7
l 5 7
l 8 6
l 6 5
l 1 5
l 6 2
l 1 6
l 2 5
l 3 8
l 4 7
l 3 5
l 1 7
l 4 6
l 2 8
l 5 8
l 6 7
l 2 3
l 1 4
```